### PR TITLE
[BUG]: Fixed Vertical & Horizontal Bar visualization issues

### PR DESCRIPTION
**Pull request by Koustubh5585**
### Description
Fixed visualization issues of vertical and horizontal bar chart
- Wrong data plotted
- Timestamp do not get plotted
- Color theme applied without selecting field from dropdown
- Tooltip options are not working
- Renamed "Metrics" to "Series" 

### Issues Resolved
https://github.com/opensearch-project/observability/issues/1153

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by mengweieric — DISMISSED** [review]

File: dashboards-observability/common/constants/shared.ts [hunk 1]
```diff
@@ -78,13 +78,13 @@ export enum VIS_CHART_TYPES {
...
-  LogsView = 'logs_view',
-  Metrics = 'metrics',
-  TableView = 'data_table',
+  // Gauge = 'gauge',
+  // Histogram = 'histogram',
+  // TreeMap = 'tree_map',
+  // Scatter = 'scatter',
+  // LogsView = 'logs_view',
+  // Metrics = 'metrics',
+  // TableView = 'data_table',
```
Comment (ps48) [inline dashboards-observability/common/constants/shared.ts:87]: can we remove comments if not needed?

Comment (mengweieric) [reply dashboards-observability/common/constants/shared.ts:87]: +1 yes let's remove these rather than commented out.

Comment (Koustubh5585) [reply dashboards-observability/common/constants/shared.ts:87]: Hi @ps48 @mengweieric commented as this is not a permanent removing of the feature, we may uncomment it in future. Anyways I will remove it.

Comment (mengweieric) [reply dashboards-observability/common/constants/shared.ts]: we can always add back without commenting or uncommenting.

**Review by joshuali925 — APPROVED** [review]

**Review by pjfitzgibbons — APPROVED** [review]

**Pull request merged**